backend/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
```

Comment:
Just a suggestion to commit this - to make things easier - happy to remove - please vote :+1: / :-1:

Reply:
I've voted no for the moment, mostly because it seems odd to have IDE specific settings in our repo & I feel like this might conflict... perhaps I'm worrying about nothing...

Reply:
I have added some VSCode settings in the FE portion of the repo 👀

Reply:
how does it interact with local/global settings I already have?

Reply:
Good point - yeah let me remove this as it may cause issues